docs/start/framework/react/migrate-from-next-js.md

```
@@ -52,23 +52,49 @@ rm postcss.config.* next.config.*
+
+```sh
+npm i @tanstack/react-router @tanstack/react-start @rsbuild/core @rsbuild/plugin-react
+```
+
+<!-- ::end:tabs -->
+
+For Tailwind CSS, install the bundler integration you want to use:
+
+<!-- ::start:tabs variant="bundler" -->
+
+# Vite
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)